evm/vm/flavors/mainnet/__init__.py

```diff
@@ -28,5 +28,8 @@
 
 MainnetEVM = EVM.configure(
     'MainnetEVM',
-    vm_block_ranges=MAINNET_BLOCK_RANGES,
+    vm_configuration=(
+        (0, FrontierVM),
```

Comment:
*maybe* go ahead and set `constants.GENESIS_BLOCK_NUMBER = 0` and then use that value here.....   or is that silly?